apps/remix-ide/src/app/tabs/locales/en/filePanel.json

```diff
@@ -29,6 +29,8 @@
   "filePanel.tssoltestghaction": "Mocha Chai Test Workflow",
   "filePanel.workspace.slitherghaction": "Adds a preset yml file to run slither analysis on github actions CI",
   "filePanel.slitherghaction": "Slither Workflow",
+  "filePanel.workspace.helperscripts": "Adds convenient scripts to the 'scripts' directory",
```

Comment:
Can you also include the translation for it?

Reply:
yes I've added the translation label.
cc @ryestew